EPI/parse-nested-GrFN.py

```diff
@@ -0,0 +1,124 @@
+import json
+from argparse import ArgumentParser
+
+
+def formatGraph(data): 
+    modelMetadata = {}
+    nodesDict = {}
+    nodes = []
+    edges = []
+
+    for key,value in data.items():
+        if key == 'variables':
+            for item in value: 
+                if 'metadata' in item:
+                    metadata = item['metadata'][0]
+                else:
+                    metadata = {}
+                item['nodeType'] = 'variable'
+                item['metadata'] = metadata
+                nodesDict[item['uid']] = item
+        if key == 'functions':
+            for item in value: 
+                if 'metadata' in item:
+                    metadata = item['metadata'][0]
+                else:
+                    metadata = {} 
+                item['nodeType'] = 'function'
+                item['metadata'] = metadata
+                nodesDict[item['uid']] = item
+        if key == 'hyper_edges':
+            for item in value: 
```

Comment:
For readability, perhaps `item` can be renamed to `hyper_edge`, or `function`, `variable` etc.?

Reply:
Yes, that's a good point, I'll just do that.